sdc/datatypes/hpat_pandas_dataframe_rolling_functions.py

```diff
@@ -78,43 +82,177 @@
 """
 
 
-def df_rolling_method_codegen(method_name, self, args=None, kws=None):
+def params2list(params):
```

Comment:
We could move this function into common_functions.py and use it in all other DF generation functions (e.g. in sdc_pandas_dataframe_reduce_columns, sdc_pandas_dataframe_apply_columns).